### PR TITLE
added my new projects

### PR DESCRIPTION
**Pull request by jayskhatri**
Added my project links

### CODE REVIEW

**Comment by ritwik12** [comment]
@jayskhatri Please check the errors.

**Review by ritwik12 — DISMISSED** [review]
Please resolve errors

**Comment by jayskhatri** [comment]
done with changes

File: _data/projects/covid19app.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+name: Covid 19 App
+desc: Covid 19 Stastics app, made in Android with JAVA
+site: https://github.com/jayskhatri/covid19-app
+tags:
+- covid19statistics
+- android
+- design
+upforgrabs:
+  name: good first issue
+  link: https://github.com/jayskhatri/covid19-app/labels/good%20first%20issue
```
Comment (shiftkey) [inline _data/projects/covid19app.yml:11]: @jayskhatri thanks for resolving the changes needed. It looks like this label doesn't have any issues assigned to it, which means it won't be listed on the site once we start retrieving stats. 

Were you interested in adding some tasks available for new contributors?

Comment (jayskhatri) [reply _data/projects/covid19app.yml:11]: Oopss sorry😬

Actually while solving previous errors i removed this label from one issue, now it's added. Thanks for pointing it out.

**Review by shiftkey — APPROVED** [review]
Thanks for the contribution @jayskhatri!

**Pull request merged**